### PR TITLE
27 sept stg release

### PR DESCRIPTION
**Pull request by kpslp**
including adding service request link. previously kellyn mentioned our SR only appears in faq

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -17,7 +17,8 @@
 - **Manage password**
   - [Reset your TechPass password](reset-password)
 - **TechPass support resources**
-  - [TechPass status](techpass-status)
+  - [TechPass status](techpass-status.md)
+  - [Raise a service request](raise-a-service-request.md)
   - [FAQs](/support/overview.md)
   - [Terms and policies](terms-and-policies)
```
Comment (JLeeChan) [inline docs/_sidebar.md:23]: how come others don't have .md?

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 ## September 2023
 
+**27 September 2023**
+
+Frontend version: 1.0.0-20230921.1438 | Backend version: 1.69.0-20230922.1123
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Storing API key for sending webhook to APIs | We are introducing the ability to store API Keys for sending webhooks to target APIs. With this feature, tenants will be able subscribe to and receive webhook events. |
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]:
```suggestion
| **Feature** | Storing API key for sending webhook to APIs | We are introducing the ability to store API Keys for sending webhooks to target APIs. With this feature, tenants will be able to subscribe and receive webhook events. |
```

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
...
+| **Feature** | Notifying tenants of expiring SAML app certificates | This new feature will notify tenants of expiring SAML app certificates. We want to provide tenants with more visibility on these expiring certificates so tenants can perform proper certificate rotation and ensure uninterrupted service. |
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]: We don't notify tenants, the notification is on the team's Slack channel, so it is only internal. This is informational for the team and don't need to appear in the release notes.

File: docs/whats-new/staging-release-notes.md [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+| **Feature** | Enhanced *user-updated* webhook to include profile changes | We have enhanced the *user-updated* webhook to include profile changes. Previously, the *user-updated* webhook was only triggered when a user's status changed. Now, in addition to status changes, tenants will be notified if other profile fields have been changed |
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md]:
```suggestion
| **Feature** | Enhanced *user-updated* webhook to include profile changes | We have enhanced the *user-updated* webhook to include profile changes. Previously, the *user-updated* webhook was only triggered when a user's status changed. Now, in addition to status changes, tenants will be notified if other profile fields have been changed. |
```

File: docs/whats-new/staging-release-notes.md (hunk 3 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md]: oo ok! it wasnt marked w a * so i assumed tenants get notified.

File: docs/_sidebar.md (hunk 1 above)
Comment (kpslp) [reply docs/_sidebar.md:23]: habit, sorry! removed. i dont think it affects anything tho

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**